test/resources/microshift-rpm.resource

```diff
@@ -107,3 +107,11 @@ Verify MicroShift RPM Install
 
     # Checks only config files from RPM packages excluding modification time check
     Command Should Work    rpm -qa microshift\* | xargs -I {} bash -c 'echo {}; sudo rpm -V --configfiles --nomtime {}'
+
+Pull Secret Should Be Installed
```

Comment:
Isnt this implied in a MicroShift start and wait? If there is no pull secret no images are pulled (unless embedded, but this is not one of those tests) and MicroShift can not start.

Reply:
Yes, it is. But be aware this keyword is executed before MicroShift is installed, so it's a pre-check to abort MicroShift installation if no pull secret is present. So I think we should keep this keyword.